src/screens/WalletScreen/index.js

```diff
@@ -49,35 +50,61 @@ type Actions = {
    * @desc Function to request wallet list update.
    */
   updateWalletList: () => void,
-}
+};
 
 type State = {
   /**
    * @desc Flag to control Refreshing on Pull to Refresh
    */
   pullToRefreshInProgress: boolean,
-}
+  /**
+   * @desc Flag to control opening of transactions Modal
+   */
+  transactionsVisible: boolean,
+  /**
+   * @desc To store transaction variables
+   */
```

Comment:
Regarding docs - maybe something like `Stores transactions modal state`. I specifically don't like word "variable" because it's not correlate with React spirit of immutability. It's not that important and there may be another opinion of course, I just wanted to share mine one

Reply:
Done